### PR TITLE
Core Team Proposal

### PR DESCRIPTION
**Pull request by mandrigin**
This PR contains a living document on Core Team Proposal.
When it is fully approved, it will be merged.

### CODE REVIEW

**Comment by oskarth** [comment]
Re maintenance: a big aspect that could probably be emphasized more is that we are an OSS project. Right now it is run more like a tech company and we aren't involving the community anywhere nearly enough. Rough sketch of what this might look like:

Core maintainers of the app and its repos (status-react, status-go, ?):
- Review/Create PRs and (bountified) issues/problem statements open to all developers
-- includes ensuring followup from PRs and issues created
- Ensure community members get help and feel heard with things like build process problems (can also get some assistance from Community here, but they generally don't code)

Role is somewhat similar to stewards/maintainers in e.g. Linux, in ideal world.

cc @mandrigin @jarradh

**Comment by mandrigin** [comment]
@oskarth that is a very good point, I will think on how to add this to the doc as well

**Comment by oskarth** [comment]
Realized I never submitted my pending review...sorry about that!

File: ideas/310-core-team-proposal.md [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+Related adjacent activities such as security, devops, marketing, QA, etc will be assumed to be provided and funded by the Gmbh until further notice.
+
+## Core participants
+
+- Adam
+- Andrea MP
+- Chad (PM)
+- DmitryS
+- Igor (tech lead)
+- Maciej (Design)
```
Comment (mandrigin) [inline ideas/310-core-team-proposal.md]: @andmironov instead

**Comment by mandrigin** [comment]
@oskarth I will take a look at your feedback today. Thanks!

File: ideas/310-core-team-proposal.md [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+
+---
+###### tags: `proposal` `core` `SNIP` `SNIP-core`
+
+**NOTE: This doc is a WIP, if you want to change something, just edit straight away. Please don't ask for permission.**
+
+# Next Actions
+
+- [ ] Potentially we can split this team in two
+    - (a) "blockchain/cryptoeconomy" team
```
[the remark this answers is not in the document]
Comment (mandrigin) [reply ideas/310-core-team-proposal.md]: yeah, that was the decision in the end

File: ideas/310-core-team-proposal.md [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+
+Desktop is out of scope, as is protocol research. Whitepaper use case implementation is partly in scope, partly not, depending on specifics and other teams.
+
+Core needs an accountable, passionate get-shit-done mentality and drive, in contrast to the meandering blame-shifting.
+
+
+## Budget
+
+Headcount-based. 7 people.
+
```
[the remark this answers is not in the document]
Comment (mandrigin) [reply ideas/310-core-team-proposal.md]: I don't know how to count. Technically, Chad, Sergey and Andrei aren't on the core team budget. Or are they? That makes us 6 developers.

File: ideas/310-core-team-proposal.md [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+4. No decent caching/reusing of artifacts (that makes the build process very slow).
+
+
+#### 6. SNT in transaction history
+
+To the Wallet team.
+
+---
+
+# Raw Data
```
[the remark this answers is not in the document]
Comment (mandrigin) [reply ideas/310-core-team-proposal.md]: done

**Pull request merged**